content/troubleshooting/scaledobject-not-scale.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Why is my `ScaledObject` paused?"
+weight = 1
++++
+
+We discovered a bug in how KEDA manages the upstream errors. By design we'd like to keep the things like they are in case of failure getting the metrics from the upstream and the `fallback` section is not defined.
+
+[We have fixed](https://github.com/kedacore/keda/pull/2604) it on v2.6.1 so after it, the new behavior is keeping the things like they are till having more information. This behavior could seem that the `ScaledObject` is paused (nothing will be done). 
+
+You can check if this is your case reviewing the logs from the KEDA pods, if you are facing with this, you should see some errors in both pods (operator and metrics server)
```

Comment:
What about this?
```suggestion
When KEDA has upstream errors to get scaler source information it will keep the current instance count of the workload unless the `fallback` section is defined.

This behavior might feel like the autoscaling is not happening, but in reality, it is because of problems related to the scaler source.

You can check if this is your case by reviewing the logs from the KEDA pods where you should see errors in both our operator and metrics server.
```